docs/examples/responses/sse_responses.py

```diff
@@ -2,15 +2,27 @@
 from typing import AsyncGenerator
 
 from litestar import Litestar, get
-from litestar.response import ServerSentEvent
+from litestar.response import ServerSentEvent, ServerSentEventMessage
 
 
-async def my_generator() -> AsyncGenerator[bytes, None]:
+async def my_generator() -> AsyncGenerator[ServerSentEventMessage, None]:
```

Comment:
```suggestion
async def my_generator() -> AsyncGenerator[Any, None]:
```
The generator is yielding many different types in the example, so perhaps this makes sense?

Reply:
what about `ServerSentEvent`?

Reply:
Is that a public type?

Reply:
don't see `_`s so yes?

Reply:
yep, forgot about this one thanks

Reply:
I don't see it in the api docs though 🤷‍♂️

Reply:
it's weird, it's not in the docs indeed but in https://github.com/litestar-org/litestar/blob/b9e501fba6ff2ab06ab3afb53d9071a0a8699c55/litestar/types/helper_types.py#L28

is there an issue with the api docs generation ?

Reply:
I guess this has to be manually added (not automatic) in https://github.com/litestar-org/litestar/blob/b9e501fba6ff2ab06ab3afb53d9071a0a8699c55/docs/reference/types.rst?

Reply:
Ref docs only include names listed in `__all__` - so I assume that's why this is.. and if this name isn't in `__all__` then that would mean it's not a public name.

We can add it though - but for the case of this example, is it really that helpful? A reader isn't going to know what the `SSEData` data type represents anyway.

Maybe we should define that type alias in the example itself, wdyt?

Reply:
But it is in `__all__` 🫠

Reply:
It seems we explicitly name things for the types namespace, and it's just not explicitly named:

https://github.com/litestar-org/litestar/blob/main/docs/reference/types.rst#L141-L150

Reply:
> I guess this has to be manually added (not automatic) in https://github.com/litestar-org/litestar/blob/b9e501fba6ff2ab06ab3afb53d9071a0a8699c55/docs/reference/types.rst?

Oops missed this, yeah, this.

Reply:
ok, so should I add it here or open another issue / pr to do that ? or revert to Any ?
I'm ok with both :)

Reply:
I would probably do it in a new PR so more eyes are on it :)